# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../git/docbook2x-texi.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- git-1.6.3.2.patch/Documentation/Makefile	2009-06-04 08:26:40.000000000 +0200
+++ git-1.6.3.2/Documentation/Makefile	2009-06-20 18:01:58.349294911 +0200
@@ -51,7 +51,7 @@
 infodir?=$(prefix)/share/info
 MAKEINFO=makeinfo
 INSTALL_INFO=install-info
-DOCBOOK2X_TEXI=docbook2x-texi
+DOCBOOK2X_TEXI=docbook2texi
 DBLATEX=dblatex
 ifndef PERL_PATH
 	PERL_PATH = /usr/bin/perl
